userguide/tutorials/catalog-examples.adoc

```diff
@@ -0,0 +1,380 @@
+|Create a subscription on `2021-09-30`   |No invoice generated
+|Trigger invoice dry run with target date `2021-09-25`   |Invoice generated for `$24.95` for `2021-09-25` to `2021-10-25`  (Second subscription not yet active)
+|Trigger invoice dry run with target date `2021-09-30`   | Invoice generated for `$20.79` for `2021-09-30` to `2021-10-25` (Prorated since BCD is 25)
+|Trigger invoice dry run with target date `2021-10-25`   | Invoice generated with two invoice items corresponding to both subscriptions for `$49.90`. 
+|===
+
+
+=== Subscription Billing Alignment
+
+When `SUBSCRIPTION` billing alignment is specified in the catalog, each subscription is invoiced separately as per its bill date. This holds true even for subscriptions in the same bundle. So, if there are multiple subscriptions associated with an account, each subscription will be invoiced separately as per its creation date irrespective of whether they are in the same subscription bundle or different subscription bundles.
+
+This example uses the  https://github.com/killbill/killbill-docs/blob/1a945188c0ef53a3c7f797b4890514d35c09c46f/catalogs/subscription-billing-alignment.xml[Subscription Billing Alignment] catalog. It specifies `SUBSCRIPTION` billing alignment.
+
+It has only one product (`Standard`). However, it has multiple plans corresponding to the `Standard` product as follows:
+
+[options="header",cols="1,1,1"]
+|===
+|Plan Name   |Product|Plan Description
+//----------------------
+|standard_monthly   |Standard   |Single EVERGREEN phase with a monthly recurring charge of $24.95   
+|standard_annual  |Standard   |Single EVERGREEN phase with an annual recurring charge of $275  
+|===
+
+
+The following table lists the actions and their results:
+
+[options="header",cols="1,1"]
+|===
+|Action   |Result   
+//-------------
+|Create a new account. Specify BCD as `25`  |Account created with BCD set to `25`
+|Create a subscription for the `standard-monthly` plan on `2021-09-17`   |Invoice generated for `$24.95` for `2021-09-17` to `2021-10-17` (BCD is ignored, subscription is billed as per its creation date)
+|Trigger invoice dry run with target date `2021-10-17`   |Invoice generated for `$24.95` for `2021-10-17` to `2021-11-17` 
+|Create a subscription for the `standard-annual` plan with date as `2021-09-30`   |No invoice generated
+|Trigger invoice dry run with target date `2021-09-30`   |Invoice generated for `$275` for `2021-09-30` to `2022-09-30` 
+|Trigger invoice dry run with target date `2021-10-17`   |Invoice generated for `$24.95` for `2021-10-17` to `2021-11-17` corresponding to `standard-monthly` plan 
+|Trigger invoice dry run with target date `2022-09-30`   |Invoice generated for `$275` for `2022-09-30` to `2023-09-30` corresponding to `standard-annual` plan  
+|===
+
+As mentioned above, even if there is an addon plan associated with a subscription, the addon plan will be invoiced as per its own date and NOT be aligned with the subscription billing date.
```

Comment:
```suggestion
As mentioned above, even if there is an add-on plan associated with a subscription, the add-on plan is invoiced as per its own date and is NOT aligned with the subscription billing date.
```

Reply:
I've removed this text and added a note at the top of this section.